### DISCUSSION TITLE
AutoGluon multiple models parallel inference for faster prediction

### DISCUSSION THREAD

**Discussion by fcoppey** [autogluon/autogluon#1770] [General]
I have 20 one-vs-rest classification models to apply on the same input, each inference takes about 0.5s. what is the best way to parallelise the 20 inferences so that I can get them all in the shortest time?

**Comment by Call-Paul** [comment]
Any solution to that problem?

**Comment by YuanfengZhang** [comment]
I'm stuck here too. Multiple autogluon instances will cause ray error resulting in all training jobs failed. But single autogluon instance is far away from fully utilizing the resources.
